doc/source/whatsnew/v1.3.0.rst

```diff
@@ -134,6 +134,30 @@ a copy will no longer be made (:issue:`32960`)
 The default behavior when not passing ``copy`` will remain unchanged, i.e.
 a copy will be made.
 
+Centered Datetime-Like Rolling Windows
```

Comment:
would not object to an example in window.rst as well

Reply:
I added the same example also to the window.rst now